blog.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Adoptadog Blog</title>
+  </head>
+  <body>
+    <header>
+      <a href="#main-content" class="skip-link">Skip to main content</a>
+      <a href="index.html"><img src="images/adoptadog-logo.svg" alt="Adoptadog logo"></a>
+      <p class="large-header-text">Seattle's premiere dog adoption agency</p>
```

Comment:
Consider using `<h1>1 header tags instead of styled paragraph tags for things like screen readers and accessibility.